db/record.c

```diff
@@ -2869,6 +2888,19 @@ static int check_blob_buffers(struct ireq *iq, blob_buffer_t *blobs,
     return 0;
 }
 
+static int check_blob_sizes(struct ireq *iq, blob_buffer_t *blobs, int maxblobs)
+{
+    for (int i = 0; i < maxblobs; i++) {
+        if (blobs[i].exists && blobs[i].length > MAXBLOBLENGTH) {
```

Comment:
Thanks Rivers.  That's what the blob[i].exists is for - that should be set to 0 if the blob ins't used.